examples/rust/dna/src/main.rs

```diff
@@ -13,7 +13,7 @@ const NUM_POINTS: usize = 100;
 
 fn main() -> Result<(), Box<dyn std::error::Error>> {
     let rec = rerun::RecordingStreamBuilder::new("rerun_example_dna_abacus")
-        .connect(rerun::default_server_addr(), rerun::default_flush_timeout())?;
+        .spawn(rerun::default_flush_timeout())?;
```

Comment:
also need to fix this up in rust.md (the tutorial)

Reply:
- See https://github.com/rerun-io/rerun/pull/4013

They are so, so many places where we state the same things.. :(